src/image/MaskedImage.cc

```diff
@@ -454,6 +454,8 @@ void processPlaneMetadata(std::shared_ptr<daf::base::PropertySet const> metadata
     }
     hdr->set("INHERIT", true);
     hdr->set("EXTTYPE", exttype);
+    hdr->set("EXTNAME", exttype);
+    hdr->set("EXTVER", 1);
```

Comment:
We are going to have to have a scheme that allows the EXTVER to be set by the caller somehow if they are writing a new set of images into an existing FITS file. I think we should probably not set EXTVER in the fits writer at all and leave it unset. That way the caller can manually set it to something and we can think later about how to explicitly control EXTVER when persisting at some other time. I'm pretty sure this all means we should not be forcing EXTVER anywhere on this PR (except maybe leaving something in the writeFits docs to say that the caller needs to set EXTVER if they are doing multiple writes to a single file).

Reply:
I removed places where EXTVER was set. As for a mention in the docs, I assume it would be in the writeFits of Exposure...? So [there](https://github.com/lsst/afw/blob/4939cbb3d9e4416b6dd5c2f40f6cf6e070342a3c/include/lsst/afw/image/Exposure.h#L358-L409)? Should I add it for every overload?

Reply:
I don't really know the best place since adding it to every `writeFits` is going to be painful. I'm not familiar with the documentation in this part of the code. Maybe @TallJimbo can advise.

Reply:
There's no good way to let the user control EXTVER that I can see; anything we'd do would be a big API change.  But I also don't see why they would need to; if they've got more than one archive in a single FITS file they're using the library incorrectly anyway.